ci-scripts/setup-translation-ll.sh

```diff
@@ -3,7 +3,7 @@ pushd ${OKTA_HOME}/${REPO}
 
 export SLACK_CHANNEL='#infodev-notifications'
 export targets=( "oce" "asa" "eu" "oie" "wf" "oag" )
-export netlify_targets=( "asa" "eu" )
+export netlify_targets=( "asa" "eu" "oag" )
```

Comment:
we're moving just the OAG target here, right? PR title says WF and OAG...

Reply:
yeah, I've updated the desciption and missed the title